src/PrimitiveObsession.Test/EngineContainerBuilderTest.cs

```diff
@@ -2,7 +2,7 @@
 using Autofac;
 using FluentAssertions;
 using PrimitiveObsession.Config;
-using PrimitiveObsession.DI;
```

Comment:
Hai chiamato la directory DI, ma secondo me non è il termine corretto. La Dependency Injection è la tecnica di iniezione delle istanze dal costruttore, ma non è quello che fa AutoFac. AutoFac è un framework di Inversion of Control, che è diverso: non sono i moduli che fanno DI, sei tu nel codice che lo fai, e AutoFac lo sfrutta.

Sicché voglio dire: la directory contiene i moduli di AutoFac modules, secondo me non va chiamata DI, al limite IoC.